docs/en/05-supported-backends/generate_backends_md_table.md

```diff
@@ -0,0 +1,34 @@
+# generate_md_table
```

Comment:
I don't think this doc should be placed in supported-backends. Any advice @RunningLeon

Reply:
Yes. Could be useful tools: https://github.com/open-mmlab/mmdeploy/blob/master/docs/en/02-how-to-run/useful_tools.md